.travis.yml

```diff
@@ -37,6 +37,13 @@ env:
     - TRAVIS_ENV="--exhaustive --folder=c/pthread"
     - TRAVIS_ENV="--exhaustive --folder=c/strings"
     - TRAVIS_ENV="--exhaustive --folder=c/special"
+    - TRAVIS_ENV="--exhaustive --folder=rust/basic --languages rust"
```

Comment:
Minor point, but to make these consistent, please change this to `--languages=rust`.